### PR TITLE
Create a new doc for Organization settings

### PR DESCRIPTION
**Pull request by jwitz**
This doc is required now that we have some very specific undocumented settings in the Organization screen, such as the environment secrets toggle. I consider the scope of this doc to be any setting that:

- Applies generally across multiple Astro entities (ie. "Create a cluster" wouldn't be here because of its specificity clusters, while "Environment secrets manager" does fit since the concept goes across user access/ Workspaces/ local development/ etc)
- Is only configurable by an Organization Owner. Billing details/ settings would need another doc.

### CODE REVIEW

File: astro/organization-settings.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Update your Organization name
+
+Your Organization name is human-readable name that appears in the Cloud UI and in the Astro CLI. Updating your Organization name is a cosmetic change that has no affect on any technical Organization data, such as your Organization ID or short name, which don't change for lifetime of the Organization. 
+
+1. In the Cloud UI, click your Workspace name in the upper left corner, then click **Organization Settings**. This opens the **General** Organization page.
+2. In the **Organization Detail** section, click **Edit Details**.
+3. Give your Organization a new **Organization Name**, then click **Update Organization**.
+
+## Configure environment secrets fetching for the Astro Environment Manager
```
Comment (paolaperaza) [inline astro/organization-settings.md:29]: Link to main connections doc here too

File: astro/organization-settings.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: 'Update settings for your Astro Organization'
+sidebar_label: 'Update Organization settings'
+id: organization-settings
```
Comment (lzdanski) [inline astro/organization-settings.md:4]: I've merged the quick fix changes, so you should be able to add a link to this page from the main connection manager page you referenced in this comment: https://github.com/astronomer/docs/pull/3181#discussion_r1426744412

File: astro/organization-settings.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Your Organization name is human-readable name that appears in the Cloud UI and in the Astro CLI. Updating your Organization name is a cosmetic change that has no affect on any technical Organization data, such as your Organization ID or short name, which don't change for lifetime of the Organization. 
+
+1. In the Cloud UI, click your Workspace name in the upper left corner, then click **Organization Settings**. This opens the **General** Organization page.
+2. In the **Organization Detail** section, click **Edit Details**.
+3. Give your Organization a new **Organization Name**, then click **Update Organization**.
+
+## Configure environment secrets fetching for the Astro Environment Manager
+
+When members of your Organization test an Astro project using the Astro CLI, they can pull connections configured in the Astro Environment manager to their local Airflow environment based on their Workspace credentials. See [Import and export Airflow connections and variables](import-export-connections-variables.md#from-the-cloud-ui) for more details.
```
Comment (jwitz) [inline astro/organization-settings.md:31]:
```suggestion
When members of your Organization test an Astro project using the Astro CLI, they can pull connections configured in the [Astro Environment Manager](#create-and-link-connections) to their local Airflow environment based on their Workspace credentials. See [Import and export Airflow connections and variables](import-export-connections-variables.md#from-the-cloud-ui) for more details. 
```

File: astro/organization-settings.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+- [Create a dedicated Astro cluster](create-dedicated-cluster.md)
+- [Create and manage Organization API tokens](organization-api-tokens.md)
+
+## Prerequisites
+
+- Organization Owner permissions
+
+## Update your Organization name
+
+Your Organization name is human-readable name that appears in the Cloud UI and in the Astro CLI. Updating your Organization name is a cosmetic change that has no affect on any technical Organization data, such as your Organization ID or short name, which don't change for lifetime of the Organization.
```
Comment (jwitz) [inline astro/organization-settings.md:23]:
```suggestion
Your Organization name is a human-readable name that appears in the Cloud UI and in the Astro CLI. Updating your Organization name is a cosmetic change that has no affect on any technical Organization data, such as your Organization ID or short name, which don't change for lifetime of the Organization. 
```

**Pull request merged**